tests/test_commands.py

```diff
@@ -5,6 +5,7 @@
 import subprocess
 import sys
 import tempfile
+import re
```

Comment:
:lipstick: Let’s keep the imports sorted alphabetically.

```suggestion
import re
import subprocess
import sys
import tempfile
```